tests/benchdnn/dnn_types.cpp

```diff
@@ -1168,9 +1196,16 @@ dnnl_primitive_attr_t create_dnnl_attr(
             } else if (e.is_binary_kind()) {
                 const auto &src1_md = attr_args.get_md(
                         (DNNL_ARG_ATTR_MULTIPLE_POST_OP(idx) | DNNL_ARG_SRC_1));
+                const auto &src2_md = attr_args.get_md(
+                        (DNNL_ARG_ATTR_MULTIPLE_POST_OP(idx) | DNNL_ARG_SRC_2));
                 assert(query_md_ndims(src1_md) != 0);
-                DNN_SAFE_V(dnnl_post_ops_append_binary(
-                        ops, e.binary.alg, src1_md));
+
+                if (e.is_binary_kind_with_ternary_op())
+                    assert(query_md_ndims(src2_md) != 0);
```

Comment:
In release mode this code should break regular post-ops as C call would be under if, unless I'm really missing something.

```suggestion
                if (e.is_binary_kind_with_ternary_op()) {
                    assert(query_md_ndims(src2_md) != 0);
                }
```